src/gmp.c

```diff
@@ -11049,16 +11049,16 @@ buffer_overrides_xml (GString *buffer, iterator_t *overrides,
 
           free (name_task);
 
-          if (include_result && override_iterator_result (overrides))
+          if (include_result && uuid_result
+              && override_iterator_result (overrides))
             {
               iterator_t results;
               get_data_t *result_get;
               result_get = report_results_get_data (1, 1,
                                                     1, /* apply_overrides */
                                                     0, /* autofp */
                                                     0  /* min_qod */);
-              result_uuid (override_iterator_result (overrides),
-                           &(result_get->id));
```

Comment:
Don't know what these two lines have been intended for. Is it ok to remove them?

Reply:
They are there to get the UUID of the result but that is already done assigning it to uuid_result near the start of the function.
The replacement line then copies uuid_result to the get data struct instead of getting it again using result_uuid.